environments/test/vars.tf

```diff
@@ -77,7 +77,7 @@ variable "mongo_count" {
 
 variable "mongo_ami" {
     description = "The ami to use for the mongodb instance. Here we're just using an Ubuntu 14.04 LTS public image."
-    default = "ami-4d202037"
+    default = "ami-07ebfd5b3428b6f4d" # 14.04 is no longer supported, going to use 18.04
```

Comment:
14.04 failed on 'apt-get update` command, because it is no longer supported and the repos have been removed. Using the 18.04 AMI instead